content/vendor/additional-objects/support-bundle.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2019-11-27
+linktitle: "Support Bundle"
+title: Support Bundle
+weight: 20050
+---
+
+The Support Bundle diagnostic tool collects and analyzes troubleshooting data from a deployed application.  The tool depends on two Custom Resources.
```

Comment:
It’s not from a deployed application, it’s from a cluster. A useful way to use the support bundle is if the application won’t deploy.